src/libraries/Microsoft.VisualBasic.Core/src/ILLinkTrim.xml

```diff
@@ -0,0 +1,6 @@
+﻿<linker>
+  <assembly fullname="Microsoft.VisualBasic.Core">
+    <!-- Required for type querying via IDispatch -->
+    <type fullname="Microsoft.VisualBasic.CompilerServices.UnsafeNativeMethods/*"/>
```

Comment:
How many types would be needed here if we list them out? It would probably be best to list the ones that need to be preserved than using a ‘*’, in case unnecessary ones are added in the future.